WcaOnRails/app/views/incidents/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= horizontal_simple_form_for(@incident, html: { class: "incident-form" }) do |f| %>
+
+  <div class="form-inputs">
+    <%= f.input :name %>
+    <%= f.input :private_description, input_html: { class: "markdown-editor" } %>
+    <%= f.input :private_wrc_decision, input_html: { class: "markdown-editor" } %>
+    <%= f.input :public_summary, input_html: { class: "markdown-editor" } %>
+    <hr/>
+    <%# FIXME improve this collection with better name/values %>
```

Comment:
It's also worth noting that this allows selecting an empty status, which probably should not be allowed:

![image](https://user-images.githubusercontent.com/277474/30075877-06add830-922c-11e7-9bd9-f9835ae43397.png)